package.json

```
@@ -43,23 +43,23 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.34",
     "@fortawesome/free-solid-svg-icons": "^5.15.2",
     "@fortawesome/react-fontawesome": "^0.1.14",
-    "@hookform/resolvers": "1.3.0",
+    "@hookform/resolvers": "1.3.4",
```

The `@hookform/resolvers` update is breaking our tests and I think our build process too, because they have a problem with their codebase, but they fix it in [this PR](https://github.com/react-hook-form/resolvers/pull/108). Can we wait until this PR is merged and keep it at `1.3.0`?
```suggestion
    "@hookform/resolvers": "1.3.0",
```